### PR TITLE
Ignore insertData when data is empty

### PR DESCRIPTION
**Pull request by dmikurube**
Just a workaround for a corner case. If a request comes with empty data `[]`, it fails.

I believe empty data would not come from the real aware-light/aware-client. Just wanted this not to fail when I test the server with `curl`.

### CODE REVIEW

File: src/main/kotlin/com/awareframework/micro/MySQLVerticle.kt [hunk 1]
```diff
@@ -218,6 +218,10 @@ class MySQLVerticle : AbstractVerticle() {
    * Insert batch of data into database table
    */
   fun insertData(table: String, device_id: String, data: JsonArray) {
+    if (data.size() <= 0) {
```
Comment (denzilferreira) [inline src/main/kotlin/com/awareframework/micro/MySQLVerticle.kt]: instead of size, could use isEmpty?

File: src/main/kotlin/com/awareframework/micro/PostgresVerticle.kt [hunk 2]
```diff
@@ -225,6 +225,10 @@ class PostgresVerticle : AbstractVerticle() {
    * Insert batch of data into database table
    */
   fun insertData(table: String, device_id: String, data: JsonArray) {
+    if (data.size() <= 0) {
```
Comment (denzilferreira) [inline src/main/kotlin/com/awareframework/micro/PostgresVerticle.kt]: could use isEmpty?

**Comment by dmikurube** [comment]
Ah, that sounds better! Addressed in ec1589aa2f107b6a00804dd5b1fd2042a22b66fb

**Review by denzilferreira — APPROVED** [review]

**Comment by dmikurube** [comment]
Thanks!
